src/app/story/storysidebar/storysidebar.component.html

```diff
@@ -1,45 +1,65 @@
 <div class="sidebar" [ngClass]="{'sidebar--open': sidebarOpen, 'sidebar--closed': !sidebarOpen}">
-  <div class="sidebar-title">
-    <a class="sidebar-title__arrow" href="#" (click)="sidebarOpen = !sidebarOpen; $event.preventDefault()">
-      <i [ngClass]="{'fa-angle-double-right': sidebarOpen, 'fa-angle-double-left': !sidebarOpen}"
-          class="fa fa-th" aria-hidden="true">
-      </i>
-    </a>
-    <ng-container *ngIf="sidebarOpen">Properties</ng-container>
-  </div>
-  <ng-container *ngIf="sidebarOpen && node">
-    <a class="sidebar__external-system-link" [href]="node | nodeExternalUrl" target="_blank">
-      <!-- *ngIf is used because it should only tooltip on long UIDs -->
-      <h2 *ngIf="(node.display_name | nodeDisplayName).length > 30; else shortUid" tooltip="{{ node.display_name | nodeDisplayName }}">
-        {{ node.display_name | nodeDisplayName | truncate: 30 }}
-      </h2>
-      <ng-template #shortUid>
-        <h2>{{ node.display_name | nodeDisplayName }}</h2>
-      </ng-template>
-    </a>
+  <div class="timeline-properties mobile-hide" (click)="sidebarOpen = !sidebarOpen; $event.preventDefault()">
```

Comment:
When this was added, the animation of the sidebar opening and closing looks off.